gcloud/start-remote-job

```diff
@@ -140,6 +147,6 @@ result = sp.run([
     # Note that if this session is already running, tmux should prevent us
     # from running it again.
     f"tmux new-session -s {args.job_name} "
-    f"{src_dir}/start-training {data_dir} --shutdown --ensure-gpu " +
-    ' '.join(remaining_args)
+    f"{src_dir}/start-training {data_dir} --ensure-gpu " + ' '.join(remaining_args)
```

Comment:
Why get rid of `--shutdown`? Do you really need the `bash` call when the training script ends in `subprocess.run("bash -il", shell=True)`?

Reply:
Removing `--shutdown` because I often don't want a shutdown, but if it's hardwired it can't be removed (it can be added back with `--shutdown`, but could also live with `--no-shutdown` on the outer script if you feel strongly).

I've found that weird things happen on the server often enough that I appreciate being dropped into a shell when `tmux` exits, and can live with logging out when it turned out to not be necessary. Could also control that with a flag if you feel strongly!

Reply:
I think having shutdown by default is good, because otherwise you run the risk of forgetting to shut down and getting a big bill from gcloud. A `--no-shutdown` flag is a good idea though. Adding that in now.

Re bash, what we really want is a way to keep the tmux session alive when something fails so that we can do debugging. That means running bash out of this script, as you have it here, but as part of the tmux command and not after it. Changing that now too.